### DISCUSSION TITLE
Combat, ranged creatures with double shot, display the potential damage with "(x2)" for the troops they shoot at

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4737] [Ideas]
For ranged creatures with double shot, I suggest to display the potential damage with "(x2)" for the troops they shoot at.
Except when the troop doesn't have enough shots left to shoot twice. :)
**Edit:** I think that displaying "(x1)" would be nice when the troop has only one ammo left whereas she could fire twice.

In fheroes2:

In the example:
"Damage: 4 - 7 (x2)"
![Ranged double shot x2 001](https://user-images.githubusercontent.com/43583994/144421154-7cf3cacc-b834-4816-b18e-bff179c2681a.png)

In the example:
"Damage: 5 (x2)"
![Ranged double shot x2 002](https://user-images.githubusercontent.com/43583994/144421188-3052f06e-af5c-4638-b4ce-f0d8d9d4d841.png)

**Edit.**

Related issues: https://github.com/ihhub/fheroes2/issues/1619, https://github.com/ihhub/fheroes2/issues/1620, https://github.com/ihhub/fheroes2/issues/1624, https://github.com/ihhub/fheroes2/issues/1625.

**Comment by zenseii** [comment]
Is that info window for how many creatures you kill an experimental option? Seems pretty nice. I think Heroes V was the first to have this.

**Reply by LeHerosInconnu** [reply]
Hello @zenseii,

> Is that info window for how many creatures you kill an experimental option? Seems pretty nice. I think Heroes V was the first to have this.

Yes, it's the "battle: show damage info" option.
![Battle show damage 01](https://user-images.githubusercontent.com/43583994/144427466-818f5225-b182-4a0d-b372-cf6bba1e8dbb.png)

It's not perfect yet, but there is progress. :)
For example, the box is cut for the troops on the right side of the screen. (https://github.com/ihhub/fheroes2/issues/1619)
![Battle show damage 02](https://user-images.githubusercontent.com/43583994/144427520-f84cbcb8-2d07-4535-8a34-4cc961047e10.png)

I don't remember for Heroes 5, I only played the demo briefly. :D

**Reply by zenseii** [reply]
Thanks! Yeah this 2x seems like a simple and unproblematic addition to make.

**Comment by LeHerosInconnu** [comment]
This is planned in my mod: https://github.com/ihhub/fheroes2/discussions/10124.
I close the discussion.
